### PR TITLE
add tabbable title

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
Adds informational text tabbable 

## Testing done
tabbed through fields with voice over active while locally testing at http://localhost:3001/education/opt-out-information-sharing/opt-out-form-0993/claimant-information

## Testing Plan
none

## Acceptance Criteria (Definition of Done)
warning should be read by screen readers when tabbing through fields 

#### Applies to all PRs
- [x] Provide [link](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/12670) to originating GitHub issue, or connected to it via ZenHub

### CODE REVIEW

File: src/applications/edu-benefits/0993/helpers.js [hunk 1]
```diff
@@ -24,3 +25,14 @@ export function transform(formConfig, form) {
     }
   });
 }
+/* eslint-disable */
+export const tabbableTitle = title => (<div
+  className="schemaform-block-header"
+  tabIndex="0">
+  <p
+    className="schemaform-block-title schemaform-block-subtitle"
+    id="root_view:optOutMessage__title">
```
Comment (cehsu) [inline src/applications/edu-benefits/0993/helpers.js]: Is there an extra underscore here?

File: src/applications/edu-benefits/0993/helpers.js [hunk 2]
```diff
@@ -24,3 +25,14 @@ export function transform(formConfig, form) {
     }
   });
 }
+/* eslint-disable */
+export const tabbableTitle = title => (<div
```
Comment (cehsu) [inline src/applications/edu-benefits/0993/helpers.js]: Are there other candidates for using this?

**Review by cehsu — APPROVED** [review]
LGTM.

File: src/applications/edu-benefits/0993/helpers.js (hunk 1 above)
Comment (rianfowler) [reply src/applications/edu-benefits/0993/helpers.js]: TBH I think we just copied what was on the generated site verbatim. Probably can clean this up- no reason to follow the generated pattern.

File: src/applications/edu-benefits/0993/helpers.js (hunk 2 above)
Comment (rianfowler) [reply src/applications/edu-benefits/0993/helpers.js]: I can't think of any. Our pure content pages are pretty well separated from our field pages. This was a rare case where we wanted a disclosure without an acknowledgment like a checkbox. 

I do think we should probably come up with a better name and make it available as a new form type though- `tabbable` is babble

**Pull request merged**